Source/Internal/fsPortableReflection.cs

```diff
@@ -50,7 +50,7 @@ public static TAttribute GetAttribute<TAttribute>(Type type)
         }
 
         public static Attribute GetAttribute(Type type, Type attributeType) {
-            return GetAttribute(type.GetTypeInfo(), attributeType);
+            return GetAttribute(type.GetTypeInfo().AsType(), attributeType);
```

Comment:
What's this? Was this broken before? Can we just have `type` instead of going from type info and back?